lte/gateway/python/magma/subscriberdb/protocols/s6a_proxy_servicer.py

```diff
@@ -127,6 +129,14 @@ def UpdateLocation(self, request, context):
 
         return ula
 
+    def PurgeUE(self, request, context):
+        self._print_grpc(request)
+        logging.warning(
+            "Purge request not implemented: {} {}".format(
```

Comment:
minor nit: use lazy formatting for logging, lte-test is failing because of this

```
W1202: 135, 12: Use lazy % formatting in logging functions (logging-format-interpolation)
```

Reply:
done